src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -53,7 +53,9 @@ private PreloadingStrategy getPreloadingStrategy(List<String> substationsIds) {
     public List<String> getSubstationsIds(UUID networkUuid, String variantId, List<Double> nominalVoltages) {
         Network network = getNetwork(networkUuid, PreloadingStrategy.COLLECTION, variantId);
         return network.getSubstationStream()
-                .filter(substation -> substation.getVoltageLevelStream().anyMatch(voltageLevel -> nominalVoltages == null || nominalVoltages.contains(voltageLevel.getNominalV())))
+                .filter(substation -> substation.getVoltageLevelStream().findAny().isEmpty() ||
+                        substation.getVoltageLevelStream().anyMatch(voltageLevel ->
+                                nominalVoltages == null || nominalVoltages.contains(voltageLevel.getNominalV())))
```

Comment:
Maybe this is better
```suggestion
                .filter(substation -> ( nominalVoltages == null && 
                                                 substation.getVoltageLevelStream().findAny().isEmpty() ) ||
                        substation.getVoltageLevelStream().anyMatch(voltageLevel ->
                                nominalVoltages == null || nominalVoltages.contains(voltageLevel.getNominalV())))
```